L1t stage1 unpacker 74x

mulhearn opened the pull request:
This is the 74X version of PR #9908

mulhearn committed:
back port new unpacker from 75x

mulhearn committed:
git user error, checkout did not delete these files, now deleting manually

mulhearn committed:
demo changes to HLT configs

cmsbuild commented:
A new Pull Request was created by @mulhearn  for CMSSW_7_4_X.

L1t stage1 unpacker 74x

It involves the following packages:

EventFilter/L1TRawToDigi
HLTrigger/Configuration
L1Trigger/L1TCommon


The following packages do not have a category, yet:

EventFilter/L1TRawToDigi

@Martin-Grunewald, @perrotta, @cmsbuild, @mulhearn, @fwyzard can you please review it and eventually sign? Thanks.
@Martin-Grunewald, @geoff-smith, @jalimena this is something you requested to watch as well.
You can sign-off by replying to this message having '+1' in the first line of your reply.
You can reject by replying  to this message having '-1' in the first line of your reply.
If you are a L2 or a release manager you can ask for tests by saying 'please test' in the first line of a comment.
@degano you are the release manager for this.
You can merge this pull request by typing 'merge' in the first line of your comment.

mulhearn commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: 3433731eb88e5dd4213f87206ec112407c085e55
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-9911/54/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-9911/54/summary.html

matz-e committed:
Fix firmware version determination.

matz-e committed:
Remove misleading warnings.

The code was changed a while back to accomodate cable splitting, unpacking
with fixed positions.  The warnings pertain to old code, where the unpacked
quantities were just pushed back onto the collection.

Martin-Grunewald commented:
Sorry, but is this PR still active or should be closed?

cmsbuild commented:
Pull request #9911 was updated. @Martin-Grunewald, @perrotta, @cmsbuild, @mulhearn, @fwyzard can you please check and sign again.

Martin-Grunewald commented:
-1
I believe this PR is superseeded by #10234 and should be closed.
In any case the changes in HLTrigger can not be done this way.
